test/integration/admin_login_test.rb

```diff
@@ -147,4 +147,29 @@ def setup
       assert page.has_content?("Order #{@order.id}")
     end
   end
+
+  test "an admin can change order status" do
+    ApplicationController.any_instance.stubs(:current_user).returns(user)
+    @non_admin = User.create(username: 'yayaya',
+                             password: 'password',
+                             first_name: 'John',
+                             last_name: 'Doe',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.